pkg/openapi/violations.txt

```diff
@@ -6,6 +6,10 @@ API rule violation: list_type_missing,github.com/weaveworks/ignite/pkg/apis/igni
 API rule violation: list_type_missing,github.com/weaveworks/ignite/pkg/apis/ignite/v1alpha2,VMStorageSpec,VolumeMounts
 API rule violation: list_type_missing,github.com/weaveworks/ignite/pkg/apis/ignite/v1alpha2,VMStorageSpec,Volumes
 API rule violation: names_match,github.com/weaveworks/ignite/pkg/apis/ignite/v1alpha1,VMSpec,CPUs
+API rule violation: names_match,github.com/weaveworks/ignite/pkg/apis/ignite/v1alpha2,ImageSpec,OCIRef
```

Comment:
Please name the field just `OCI` and these violations will go away.

Reply:
Done in 9d6ab9e.